telluric/georaster.py

```diff
@@ -736,6 +758,19 @@ def height(self):
         """Raster height."""
         return int(self.shape[1])
 
+    @property
+    def blockshapes(self):
+        """Raster all bands block shape."""
+        if self._blockshapes is None:
```

Comment:
What happens when we don't have blockshapes because our raster is completely in memeory? blockshapes is a feature of rasterio